src/components/dialogHelper/dialogHelper.js

```diff
@@ -106,15 +106,6 @@ import '../../assets/css/scrollstyles.css';
                     tryRemoveElement(dlg);
                 }
             }
-
-            //resolve();
-            // if we just called history.back(), then use a timeout to allow the history events to fire first
-            setTimeout(() => {
```

Comment:
Why was this removed?

Reply:
I was seeing tons of unhandled exceptions for dialogs and traced it back to this weird section of code, and removing it had no ill effects. History is already broken on dialogs in stable and I didn't want to look into it too much, so I figured it was fine.